ci/kubetest/test_clickhouse_backup.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+import time
+
+from helpers.clickhouse import get_clickhouse_cluster_service_spec
+from helpers.utils import cleanup_k8s, cleanup_helm, helm_install, wait_for_pods_to_be_ready, install_chart
+from helpers.utils import (
+    NAMESPACE,
+    cleanup_helm,
+    cleanup_k8s,
+    exec_subprocess,
+    install_chart,
+    install_custom_resources,
+    merge_yaml,
+    wait_for_pods_to_be_ready,
+)
+
+
+
+from typing import Optional
+
+VALUES_WITH_BACKUP = """
+clickhouse:
+  backup:
+    enabled: true
+    backup_schedule: "*/2 * * * *"
+    env:
+    - name: LOG_LEVEL
+      value: "debug"
+    - name: ALLOW_EMPTY_BACKUPS
+      value: "true"
+    - name: API_LISTEN
+      value: "0.0.0.0:7171"
+    # INSERT INTO system.backup_actions to execute backup
+    - name: API_CREATE_INTEGRATION_TABLES
+      value: "true"
+    - name: BACKUPS_TO_KEEP_REMOTE
+      value: "3"
+    # change it for production S3
+    - name: REMOTE_STORAGE
+      value: "s3"
+    - name: S3_ACL
+      value: "private"
+    - name: S3_ENDPOINT
+      value: http://s3-backup-minio:9000
+    - name: S3_BUCKET
+      value: clickhouse
+    - name: S3_PATH
+      value: backup
+    - name: S3_ACCESS_KEY
+      value: backup-access-key
+    - name: S3_SECRET_KEY
+      value: backup-secret-key
+    - name: S3_FORCE_PATH_STYLE
+      value: "true"
+    # remove it for production S3
```

Comment:
Same here

Reply:
this is not typo, I took it from owner of backup script, if you think we don't need, then I can remove it

Reply:
I think this answer might be for my comment below? Anyway, looks like the upstream README has the typo. The code doesn't have it https://github.com/AlexAkulov/clickhouse-backup/blob/c84824c09eb49def7a7852f7d15212246020413e/pkg/config/config.go#L93

Reply:
I've created a patch upstream https://github.com/AlexAkulov/clickhouse-backup/pull/448

Reply:
good catch,  I thought you talking about comment "# remove it for production S3"